caf-audit-monkey/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2015-2017 Hewlett Packard Enterprise Development LP.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Set root logger level and its only appender to A1.
+log4j.rootLogger="${env:CAF_LOG_LEVEL:-DEBUG}", A1
```

Comment:
Does this log4j.properties work when it comes to controlling the level of logging through and environment variable? I only ask because I recall Adam requiring an XML to do something similar for the audit web service recently?